### PR TITLE
Add squeeze distribution to i386 archicture list

### PR DESCRIPTION
**Pull request by mfvescovi**

### CODE REVIEW

File: distributions/i386 [hunk 1]
```diff
@@ -48,6 +48,12 @@ extramirrors: deb http://security.debian.org/ wheezy/updates main contrib non-fr
               deb http://ftp.debian.org/debian wheezy-updates main contrib non-free
               deb http://ftp.debian.org/debian wheezy-backports main contrib non-free
 
+[squeeze]
```
Comment (dktrkranz) [inline distributions/i386]: This should be squeeze-lts

File: distributions/i386 [hunk 2]
```diff
@@ -48,6 +48,12 @@ extramirrors: deb http://security.debian.org/ wheezy/updates main contrib non-fr
...
+suite: squeeze
+mirror: http://ftp.debian.org/debian
+components: main contrib non-free
+extramirrors: deb http://security.debian.org/ squeeze/updates main contrib non-free
```
Comment (dktrkranz) [inline distributions/i386]: No longer needed (see https://wiki.debian.org/LTS/Using)

File: distributions/i386 [hunk 3]
```diff
@@ -48,6 +48,12 @@ extramirrors: deb http://security.debian.org/ wheezy/updates main contrib non-fr
...
+              deb http://ftp.debian.org/debian squeeze-updates main contrib non-free
```
Comment (dktrkranz) [inline distributions/i386]: s/updates/lts/

**Comment by mfvescovi** [comment]
Fixed now.

**Pull request merged**